libs/librepcb/projecteditor/boardeditor/boardeditor.cpp

```diff
@@ -88,6 +88,8 @@ BoardEditor::BoardEditor(ProjectEditor& projectEditor, Project& project)
     mProject(project),
     mUi(new Ui::BoardEditor),
     mGraphicsView(nullptr),
+    mLastSearch(""),
+    mNextDeviceIndex(1),
```

Comment:
Although probably it doesn't really matter, wouldn't an initialization value `0` be more intuitive? ;)

Reply:
It should probably be 0. I don't know anymore why I chose 1